base/operators.jl

```diff
@@ -5,9 +5,18 @@
 """
     <:(T1, T2)
 
-Subtype operator: returns `true` if and only if all values of type `T1` are
+Subtype predicate: returns `true` if and only if all values of type `T1` are
 also of type `T2`.
 
+Julia is sometimes not able to prove a subtyping relation that would ideally
+hold between two types. When that happens, `<:` returns `false`.
+
+The `<:` predicate is expected (modulo bugs) to behave as a partial order. I.e.,
```

Comment:
Does it really make sense to hedge against bugs here? Surely, any and all documented behaviour ever are always "if there isn't any bugs"